orm.go

```diff
@@ -753,7 +754,19 @@ func OptFieldTypeSet(cacheType CacheType, cacheSize int) FieldOption {
 }
 
 // OptFieldTypeInt adds an integer field.
-func OptFieldTypeInt(min int64, max int64) FieldOption {
+// No arguments: min = min_int, max = max_int
+// 1 argument: min = limit[0], max = max_int
+// 2 or more arguments: min = limit[0], max = limit[1]
+func OptFieldTypeInt(limits ...int64) FieldOption {
```

Comment:
This is a clever way to preserve existing code that was using the old form. Nicely done.